sklearn/feature_selection/_rfe.py

```diff
@@ -679,6 +686,12 @@ class RFECV(RFE):
         n_features : ndarray of shape (n_subsets_of_features,)
             Number of features used at each step.
 
+        ranking(k) : ndarray of shape (n_subsets_of_features,)
+            The cross-validation ranking across (k)th fold.
+
+        support(k) : ndarray of shape (n_subsets_of_features,)
+            The cross-validation supports across (k)th fold.
```

Comment:
would be nice to update this part of the docstring to mention when each key would exist here, since it depends on constructor arguments.

Also, the `.. versionadded` directives need to be there for each newly added key.

Reply:
For the version, I didn't find anything in the docs (cf [research](https://scikit-learn.org/dev/search.html?q=versionadded)). I wrote down a version by myself, but i'm unsure as I guess it's something automated?

Reply:
Furthermore, it seems that it doesn't appear anyway. If I take the example of n_features in the dict cv_results_, it doesn't appear in [docs](https://scikit-learn.org/stable/modules/generated/sklearn.feature_selection.RFECV.html), while it's in the [code](https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/feature_selection/_rfe.py#L670).  
![Capture d’écran du 2024-11-26 16-19-26](https://github.com/user-attachments/assets/72afabe7-5b89-4b5c-86ea-ae6f8dc59f5b)

Reply:
> would be nice to update this part of the docstring to mention when each key would exist here, since it depends on constructor arguments.

I don't get this, it should appear all the time in the dict cv_results_ when RFECV is called, just as "split(k)_test_score" since it's built with the same process?